### PR TITLE
Add founder permissions on Elements

### PR DESCRIPTION
**Pull request by DeltaQuad-DQ**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -3270,6 +3270,11 @@
 		'+elementswiki' => array(
 			'founder' => array(
 				'read' => true,
+				'globalblock-whitelist' => true,
+				'interwiki' => true,
```
Comment (Reception123) [inline LocalSettings.php]: Interwiki is not given to userrights on wikis as it edits Miraheze's global interwiki.

File: LocalSettings.php [hunk 2]
```diff
@@ -3270,6 +3270,11 @@
...
+				'editusercss' => true,
+				'edituserjs' => true,
+				'siteadmin' => true,
```
Comment (Reception123) [inline LocalSettings.php]: Not sure what this permission is/does.

Comment (JohnFLewis) [inline LocalSettings.php]: Locking/unlocking a database.

**Comment by DeltaQuad-DQ** [comment]
Sorry, interwiki should have been editinterface. The siteadmin permission is a direct way of doing indefinite read-only, rather than having to go through a new PR every time. I don't expect to need to use it frequently, but it is one of those things that it would be handy to have around, just in case.

**Comment by Reception123** [comment]
@DeltaQuad-DQ From what I know you can't have siteadmin, but I am not completely sure of that. If you want this PR merged now please remove it and you may make another one for siteadmin and another sysadmin will say if you can have it or not.

**Comment by DeltaQuad-DQ** [comment]
I have made a Phabricator task regarding the siteadmin permission

**Pull request merged**